### PR TITLE
Dagger 2.11 compatible generators for Fragment modules (Kotlin&Java)

### PR DESCRIPTION
**Pull request by bhm**

### CODE REVIEW

**Comment by MasterAccountIFM** [comment]
--- JIRA LINKS ---

File: android-studio-plugin/dagger_211/InFullMvpFragmentJava_dagger211/root/src/mvp_folder/di/Module.java.ftl [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+import dagger.Provides;
+
+@Module
+public abstract class ${moduleClass} {
+
+    private ${fragmentClass} fragment;
+
+    @${scopeClass}
+    @Binds
+    abstract ${moduleClass} providesFragment(${fragmentClass} fragment)
```
Comment (PawelSwietochowski) [inline android-studio-plugin/dagger_211/InFullMvpFragmentJava_dagger211/root/src/mvp_folder/di/Module.java.ftl]: Missing ; at the end

File: android-studio-plugin/dagger_211/InFullMvpFragmentJava_dagger211/root/src/mvp_folder/di/Component.java.ftl [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+import ${relativePackage}.di.${scopeClass};
+import dagger.Subcomponent;
+import dagger.android.AndroidInjector;
+
+@${scopeClass}
+@Subcomponent
+public interface ${componentClass} {
+
+     @Subcomponent.Builder
+     public abstract class Builder extends AndroidInjector.Builder<${fragmentClass}>(){}
```
Comment (PawelSwietochowski) [inline android-studio-plugin/dagger_211/InFullMvpFragmentJava_dagger211/root/src/mvp_folder/di/Component.java.ftl]: This file is completely messed up on my pc. Do we need () here?

**Comment by PawelSwietochowski** [comment]
Tested on my windows pc.

**Comment by bhm** [comment]
Done.

I could swear I've done these `;` clean ups already, but apparently I did not.

**Pull request merged**